.github/dependabot.yml

```
@@ -0,0 +1,15 @@
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    labels:
+      - "🤖 Dependencies"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)